pkg/controller/operators/openshift/helpers.go

```diff
@@ -105,8 +105,12 @@ func (s skew) String() string {
 	if s.err != nil {
 		return fmt.Sprintf("%s/%s has invalid %s properties: %s", s.namespace, s.name, MaxOpenShiftVersionProperty, s.err)
 	}
-
-	return fmt.Sprintf("%s/%s is incompatible with OpenShift minor versions greater than %s", s.namespace, s.name, s.maxOpenShiftVersion)
+	// is it safe to ignore the error here, with the assumption
+	// that we build a skew object only after verifying that the
+	// version string is parseable safely.
+	maxOCPVersion, _ := semver.ParseTolerant(s.maxOpenShiftVersion)
+	nextY := nextY(maxOCPVersion).String()
+	return fmt.Sprintf("ClusterServiceVersions blocking upgrade to %s or higher. The maximum supported OCP version for %s/%s is %s", nextY, s.namespace, s.name, s.maxOpenShiftVersion)
```

Comment:
Am I reading things correctly that this message is printed for each CSV that is blocking the upgrade. If so, it would be duplicative to repeatedly state in our Upgradeable=False message that "ClusterServiceVersions are blocking minor version upgrades to X.Y+1 or higher"

WDYT about the final message looking something like this?
```yaml
conditions:
- type: Upgradeable
  status: False
  reason: IncompatibleOperators
  message: |
    ClusterServiceVersions are blocking minor version upgrades to X.Y+1 or higher.
      - openshift-operators/foo is supported through X.Y
      - openshift-operators/bar is supported through X.Y
      - openshift-operators/baz is supported through X.Y
```

Reply:
Ah good catch. Updated the PR: got the exact format you mentioned, [see this](https://go.dev/play/p/QZvCpbQzlbM)